### DISCUSSION TITLE
Every time I Click in a line in the the zed editor, there is an annoying suggestion pop-up comes up. I want to disable this.

### DISCUSSION THREAD

**Discussion by TanvirMahfuz** [zed-industries/zed#15767] [Feature Requests]
![Screenshot 2024-08-04 at 5 32 55 PM](https://github.com/user-attachments/assets/f4ffd099-bd38-442a-8184-f93cc10e1fd1)

**Comment by cv4x** [comment]
Sounds like the "hover_popover_enabled" setting.

Try adding this to your settings file:
```json
  "hover_popover_enabled": false,
```

You can find comments explaining the various configuration options in the default settings file. (Command Palette -> open default settings). I found this potential solution by opening that file and searching for the word "popover".

**Reply by ncpa0cpl** [reply]
this completely disables this feature right? I don't really want to get rid of it, but it's really annoying when it pops-up as I type. right now I find myself constantly mashing Esc key to see the code it's covering, but I don't want to completely disable it as it's useful to be able to hover a mouse-over a symbol and see the info on things

**Reply by cv4x** [reply]
Yeah, it would.

Hmm, do you maybe have `"auto_signature_help": true`? I found that setting caused the popover to never close when working in JS/TS. `"show_signature_help_after_edits": true` may also be related, but I didn't personally notice any unwanted behavior with that one.

**Reply by ncpa0cpl** [reply]
>Hmm, do you maybe have "auto_signature_help": true?

no, I have a very minimal config right now, only things I've changed are font size, font family and the theme. I will try changing `show_signature_help_after_edits` and will see if it helps

**Comment by aretrace** [comment]
Actual Answer
`"show_signature_help_after_edits": false`
